src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.ts

```diff
@@ -312,6 +321,46 @@ export class BuilderNavbarComponent implements OnDestroy {
     this.store.cancelSubmission();
   }
 
+  /**
+   * Create a new changelog for the active learning object
+   */
+  createChangelog() {
+    if (this.changelog) {
+      this.changelogService.createChangelog(this.store.learningObjectEvent.getValue().author.id,
+        this.store.learningObjectEvent.getValue().id,
+        this.changelog)
+          .then(() => {
+          this.showSubmission = false;
+          this.page = 1;
+        }).catch(e => {
+          console.error(e);
+          this.showSubmission = false;
+          this.page = 1;
+          this.toasterService.notify('Error!', e, 'bad', 'far fa-times');
```

Comment:
This needs to have a descriptive error message for the user.

Reply:
What descriptive error message should be displayed?  While looking up similar toaster service use for error handling, it ranges from what is displayed here and a actual message.

Reply:
Something that describes what happened to the user, and informs them what action (if any) they should take. This is a good post on error messages: https://uxplanet.org/how-to-write-good-error-messages-858e4551cd4